test/__snapshots__/index.spec.js.snap

```diff
@@ -1297,6 +1309,12 @@ Array [
     "rule": "array-bracket-spacing",
     "severity": 2,
   },
+  Object {
```

Comment:
This seems to be a rule related to `block-spacing` and not indent. Check `style.js#18`